src/behaviours/DragAndDrop/DragPreview.js

```diff
@@ -55,13 +56,23 @@ export default class DraggablePreview {
 
   render() {
     this.node.setAttribute('style', styles(this.x, this.y));
+
+    if (this.validMove) {
+      this.node.setAttribute('class', 'draggable-preview');
+    } else {
+      this.node.setAttribute('class', 'draggable-preview node--inactive');
```

Comment:
Should this be: `draggable-preview--obstacle` instead of `node--inactive`?

I know that we currently (and will probably always) only add this HOC to nodes, but it isn't intrinsically node specific.

Reply:
Good point. I can make that change in another branch since this one was merged already.